Missing dependency on ament_index_cpp in visp_bridge (ROS 2 Jazzy)

Hello,

I'm currently working with your packages on **ROS 2 Jazzy**, and I encountered a build error when compiling the `visp_bridge` package:

```
t_ws/src/vision_visp/visp_bridge/src/path_retriever.cpp:47:10: fatal error: ament_index_cpp/get_package_prefix.hpp: No such file or directory
   47 | #include <ament_index_cpp/get_package_prefix.hpp>
```

This issue occurs because the package is missing a dependency on `ament_index_cpp`.

To resolve the problem, I added `ament_index_cpp` to both the `CMakeLists.txt` and the `package.xml` of `visp_bridge`. After doing this, the build completed successfully.

I have created a pull request with the fix

--- a/visp_bridge/CMakeLists.txt
+++ b/visp_bridge/CMakeLists.txt
@@ -15,6 +15,7 @@ find_package(geometry_msgs REQUIRED)
 find_package(sensor_msgs REQUIRED)
 find_package(camera_calibration_parsers REQUIRED)
 find_package(VISP REQUIRED COMPONENTS visp_core)
+find_package(ament_index_cpp REQUIRED)
 
 # add library 
 add_library(${PROJECT_NAME} src/3dpose.cpp src/camera.cpp src/image.cpp src/path_retriever.cpp)
@@ -24,6 +25,7 @@ ament_target_dependencies(${PROJECT_NAME}
   rclcpp
   sensor_msgs
   VISP
+  ament_index_cpp
 )
 
 include(GenerateExportHeader)

--- a/visp_bridge/package.xml
+++ b/visp_bridge/package.xml
@@ -22,6 +22,7 @@
   <build_depend>sensor_msgs</build_depend>
   <build_depend>visp</build_depend>
   <build_depend>camera_calibration_parsers</build_depend>
+  <build_depend>ament_index_cpp</build_depend>
 
   <export>
     <build_type>ament_cmake</build_type>

